more_modbus/RegisterMapping.h

```diff
@@ -147,12 +153,14 @@ class RegisterMapping
      * @param deadbandValue indicates a change in value of the register that is insignificant data
      * @param frequencyFilterValue changes that occur within the given time (in milliseconds) that will be ignored
      * @param repeatedWrite The minimal time between two writes for a mapping.
+     * @param defaultValue The default value of the register.
      */
     RegisterMapping(std::string reference, RegisterType registerType, std::vector<int32_t> addresses, OutputType type,
                     OperationType operation, bool readRestricted = false, int16_t slaveAddress = -1,
                     double deadbandValue = 0.0,
                     std::chrono::milliseconds frequencyFilterValue = std::chrono::milliseconds(0),
-                    std::chrono::milliseconds repeatedWrite = std::chrono::milliseconds{0});
+                    std::chrono::milliseconds repeatedWrite = std::chrono::milliseconds{0},
+                    std::string defaultValue = "");
```

Comment:
Does this need to be a constructor argument since its never passed to any constructor call?